nemo/collections/nlp/parts/nlp_overrides.py

```diff
@@ -985,3 +986,34 @@ def _fetch_next_batch(self, iterator: Iterator) -> None:
             assert isinstance(dataloader, Sized)  # `_has_len` is True
             self.done = self.fetched >= len(dataloader)
         self.on_fetch_end(batch, start_output)
+
+
+class CustomProgressBar(TQDMProgressBar):
+    """
+    Add a CustomProgressBar to remove 's/it' and add train_step_timing for megatron models
+    """
+
+    def __init__(self, cfg):
+        super().__init__()
+        self.cfg = cfg
+
+    def init_train_tqdm(self):
+        """
+        Override progress bar total to reflect total microbatches as PTL 2.0 progress bar indicates total microbatches, and bar format 
```

Comment:
> reflect total microbatches as PTL 2.0 progress bar indicates total microbatches

is this a typo?

Reply:
@aklife97 thanks for the review. Sorry I din't get what typo you are referring to here ?